src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -542,6 +547,19 @@ internal ManagedCodePatterns(ManagedCodeConventions conventions)
                         {
                             new PatternDefinition("embed/{tfm}/{assembly}", table: DotnetAnyTable),
                         });
+
+                MSBuildTransitiveFiles = new PatternSet(
+                    conventions.Properties,
+                    groupPatterns: new PatternDefinition[]
+                    {
+                        new PatternDefinition("buildTransitive/{tfm}/{msbuild?}", table: DotnetAnyTable),
+                        new PatternDefinition("buildTransitive/{msbuild?}", table: null, defaults: DefaultTfmAny)
+                    },
+                    pathPatterns: new PatternDefinition[]
+                    {
+                        new PatternDefinition("buildTransitive/{tfm}/{msbuild}", table: DotnetAnyTable),
```

Comment:
We should probably be using any table for both of these. 
DotnetAnyTable just applies any to netframework, but that's a separate issue I guess.